java/lang/Double/Double.cpp

```diff
@@ -342,58 +239,58 @@ int Double::compareTo(Double anotherDouble) {
 }
 
 long Double::doubleToLongBits(double valueDouble) {
-    if (isNaN(valueDouble) || isInfinite(valueDouble))
-        return -9999;
-    long doubleToLongBitsResult = doubleToRawLongBits(valueDouble);
-
-    // Check for NaN based on values of bit fields, maximum
-    // exponent and nonzero significand.
-
-    long andOperatorBetweendoubleToLongBitsResultAndEXP_BIT_MASK
-            = doubleToLongBitsResult & EXP_BIT_MASK;
+    return doubleToRawLongBits(valueDouble);
+}
 
-    long andOperatorBetweendoubleToLongBitsResultAndSIGNIF_BIT_MASK
-            = doubleToLongBitsResult & SIGNIF_BIT_MASK;
+long Double::doubleToRawLongBits(double doubleInput) {
+    string doubleInputToBinary64StringType;
+    long resultDoubleToRawLongBits;
+    int tempValue ;
+    int exponent;
 
-    boolean isEqualEXP_BIT_MASK
-            = (andOperatorBetweendoubleToLongBitsResultAndEXP_BIT_MASK == EXP_BIT_MASK);
+    doubleInputToBinary64StringType = doubleToBinary64StringType(doubleInput);
 
-    boolean isNotEqual0L = (andOperatorBetweendoubleToLongBitsResultAndSIGNIF_BIT_MASK != 0L);
+    exponent = 62;

```

Comment:
I do that to have easier looking with my code, when I use the variable, I assign value to them, and I must not to check or remember their original value.